src/test/daml/ContingentClaims/Test/Lifecycle.daml

```diff
@@ -464,8 +464,10 @@ testFloatingRateNote = script do
     fixingDate = date 2022 Mar 05
     paymentDate = date 2022 Mar 10
 
+    -- CREATE_FRN_USING_OBSERVE_BEGIN
```

Comment:
I am not sure we should advertise this way of using `Observe` when this could be written more simply with `ObserveAt`.

The case where we should advertise `Observe` is when `fixingDate == paymentDate`.

What do you think?